README.md

```diff
@@ -200,6 +200,58 @@ pipenv shell
 apt install lintian
 make test
 ```
+## Signing sources
+
+SecureDrop Workstation code spans across the following repositories:
+
+https://github.com/freedomofpress/securedrop-client
+https://github.com/freedomofpress/securedrop-debian-packaging
+https://github.com/freedomofpress/securedrop-proxy
+https://github.com/freedomofpress/securedrop-sdk
+https://github.com/freedomofpress/securedrop-workstation
+https://github.com/freedomofpress/qubes-template-securedrop-workstation
+
+
+### Release
+1. For each release, a tag will be signed in all the above repos.
+
+2. Create a Makefile target in securedrop-debian-packaging repo that contains release tags / commit hashes for each repository used for the release. To verify the tag signature and check out the packaging logic:
+```
+git tag -v <tag>
+git checkout <tag>
+```
+
+3. Metadata (e.g. commit hash for release) should be tracked inside the .deb (e.g.: `/usr/share/packagename/release-info.txt`)
+
+### Signing binaries/packages
+
+#### Debs
+Apt repository Release file will be signed, containing checksum of the debs.
+
+#### Rpms
+The entire RPM must be signed. This process also requires a Fedora machine/VM on which the GPG signing key (either in GPG keyring or in qubes-split-gpg) is setup.
+
+`rpm -Kv` indicates if digests and sigs are OK. Before signature it should not return signature, and `rpm -qi <file>.rpm` will indicate an empty Signature field.
+
+Set up environment:
+`sudo dnf install rpm-build /usr/bin/rpmsign`
+Set your vault vm in /rw/config/gpg-split-domain
+Edit ~/.rpmmacros with the following contents:
+```
+%_signature gpg
+%_gpg_name <gpg_key_id>
+%__gpg /usr/bin/qubes-gpg-client-wrapper
+%__gpg_sign_cmd %{__gpg} qubes-gpg-client-wrapper --no-verbose --detach-sign %{__plaintext_filename} --output %{__signature_filename}
+```
+
+Sign the rpm:
+`rpm --resign <rpm>.rpm` (--addsign would allow us to apply multiple signatures to the RPM)
+Verify that the rpm is signed:
+`rpm -qi<file.rpm>` should now show that the file is signed
+`rpm -Kv` will complain that signature is not ok (`Digests SIGNATURES NOT OK`)
+This is because the rpm key needs to be added locally:
+`sudo rpm --import <publicKey>.asc`
+`rpm -Kv` will now say signatures are ok (`Digests signatures OK`)
```

Comment:
Have not run these commands yet, but the phrase "the rpm key needs to be added locally" is confusing. Given that the `~/.rpmmacros` file contains an override to the gpg binary (according to definitions at http://rpm5.org/docs/api/config_macros.html), the keys should be available for verification, no? A bit more language here explaining the unintuitive behavior is warranted—particularly what "rpm key" refers to—or, better, we can hopefully resolve it.

The entire verify/sign workflow would perhaps best be provided as a script, though I recognize ferrying the RPM to be signed, as well as the signing logic, between AppVMs is going to be painful.